Packs/Unit42ThreatIntelligencebyPaloAlto/Integrations/Unit42Intelligence/README.md

```diff
@@ -0,0 +1,322 @@
+Enrich indicators with Unit 42 threat intelligence data including verdicts, tags, and relationships.
+
+## Configure Unit 42 Intelligence in Cortex
+
+| **Parameter** | **Description** | **Required** |
+| --- | --- | --- |
+| Server URL | The base URL for the Unit 42 Intelligence API | True |
```

Comment:
@MosheEichler , this needs to be generated again by SDK as it isn't up to date (Server URL integration param was removed)